packages/redux-dynamic-modules-react/src/DynamicModuleLoader.tsx

```diff
@@ -118,60 +79,73 @@ class DynamicModuleLoaderImpl extends React.Component<
         }
     }
 
-    private _addModules(): void {
-        const { createStore, modules } = this.props;
-        if (!this._store) {
-            if (createStore) {
-                this._store = createStore();
-                this._providerInitializationNeeded = true;
-            } else {
-                throw new Error(
-                    "Store could not be resolved from React context"
+    public render(): React.ReactNode {
+        if (this.state.readyToRender) {

```

Comment:
comments?

Reply:
What comments are you looking for here? This seems pretty self explanatory to me